src/plugin.js

```diff
@@ -97,28 +90,6 @@ class NpmInstallPlugin {
     }
   }
 
-  preCompile(compilation, next) {
-    if (!this.preCompiler) {
-      const { options } = this.compiler;
-      const config = Object.assign(
-        // Start with new config object
-        {},
-        // Inherit the current config
-        options,
-        {
-          // Register plugin to install missing deps
-          plugins: [new NpmInstallPlugin(this.options)],
-        }
-      );
-
-      this.preCompiler = webpack(config);
-      this.preCompiler.outputFileSystem = createFsFromVolume(new Volume());
-      this.preCompiler.outputFileSystem.join = path.join.bind(path);
```

Comment:
We should keep it otherwise `webpack serve will not work`

Reply:
can you clarify(maybe I misunderstood), how will `webpack serve` get affected?

Reply:
When we use `webpack server` we use memory memory fs so install packages will not work

Reply:
but this is unused for some reason 🤔

Reply:
Because we don't have test with webpack-dev-server, we need to add them